extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -229,30 +258,35 @@ protected function prepValuesFromHtml( $html ) {
 			$plaintext = $this->getPlaintextFromDom( $dom );
 			$paragraphs = $this->getParagraphsFromDom( $dom );
 		} else {
-			$plaintext = html_entity_decode( strip_tags( $html ), ENT_COMPAT, 'UTF-8' );			
+			$plaintext = html_entity_decode( strip_tags( $html ), ENT_COMPAT, 'UTF-8' );
 		}
-		$plaintext = trim(preg_replace(static::SPACE_SEQUENCE_REGEXP, ' ', $plaintext));
-		$paragraphString = trim(preg_replace( static::SPACE_SEQUENCE_REGEXP, ' ', implode( ' ', $paragraphs ) )); // can be empty
-		$words = preg_split( '/ /', $paragraphString?: $plaintext);
+		$plaintext = trim( preg_replace( static::SPACE_SEQUENCE_REGEXP, ' ', $plaintext ) );
+		$paragraphString =
+			trim( preg_replace( static::SPACE_SEQUENCE_REGEXP, ' ', implode( ' ', $paragraphs ) ) ); // can be empty
+		$words = preg_split( '/ /', $paragraphString ?: $plaintext );
 		$wordCount = count( $words );
-		$upTo100Words = implode( ' ', array_slice( $words, 0, min( array( $wordCount, 100 ) ) ) );
+		$upTo100Words = implode( ' ', array_slice( $words, 0, min( [ $wordCount, 100 ] ) ) );
 		$this->pushNolangTxt( $upTo100Words );
 
-		return  array_merge( $result,
-				[
-				'nolang_txt'           => $upTo100Words,
-				'words'                => $wordCount,
+		return array_merge(
+			$result,
+			[
+				'nolang_txt' => $upTo100Words,
+				'words' => $wordCount,
 				$this->field( 'html' ) => $plaintext
-				]);
+			]
+		);
 	}
 
 	/**
 	 * Assigns infobox-based values to result (passed by reference), when found.
+	 *
 	 * @param simple_html_dom $dom
+	 *
 	 * @return array
 	 */
 	protected function extractInfoboxes( simple_html_dom $dom ) {
-		$result = array();
+		$result = [];
 		$infoboxes = $dom->find( 'table.infobox,table.wikia-infobox' );
 		if ( count( $infoboxes ) > 0 ) {
 			$result['infoboxes_txt'] = [];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "infoboxes_txt" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)